README.md

```diff
@@ -141,15 +141,15 @@ It is recommended that no work start on implementation until an RFC is accepted.
 [The RFC life-cycle]: #the-rfc-life-cycle
 
 Once an RFC becomes "active" then authors may implement it and submit the
-feature as a pull request to the OBS Porject (or appropriate module) repo. Being 
+feature as a pull request to the OBS Project (or appropriate module) repo. Being
 "active" is not a rubber stamp, and in particular still does not mean the feature
 will ultimately be merged; it does mean that in principle all the major interested
 parties have agreed to the feature and are amenable to merging it.
 
 Furthermore, the fact that a given RFC has been accepted and is "active"
 implies nothing about what priority is assigned to its implementation, nor does
-it imply anything about whether an OBS Project developer has been assigned the 
-task of implementing the feature. While it is not *necessary* that the author 
+it imply anything about whether an OBS Project developer has been assigned the
+task of implementing the feature. While it is not *necessary* that the author
 of the RFC also write the implementation, it is by far the most effective way
```

Comment:
I assumed `should` was omitted, I didn't fix `write` to `writes`.

Reply:
I can see why spellcheck would want to make that change, but this reads correctly to me as a native english speaker. Adding "should" makes it read more awkward, IMO.

Reply:
> While it is not *necessary* that the author of the RFC also write the implementation

This is grammatically correct, as far as I can tell.